o8g/Sets/eab246c4-1270-4d99-b855-e26f21fbdfee/set.xml

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<set
+  name="Tokens"
+  id="eab246c4-1270-4d99-b855-e26f21fbdfee"
+  gameId="bb0f02e7-2a6f-4ae3-84a2-c501b4176844"
+  gameVersion="0.0.1.0"
+  version="1.2"
+  hidden="true">
+  <markers>
+    <marker id="054b9f42-ac6a-4cc2-9228-294d5df5fb7e" name="Dishonored"/>
+    <marker id="1c469258-900d-44e7-b005-d3c5d0de3f95" name="Fate"/>
+    <marker id="af5fc343-121e-4ecc-9d21-f3886644b473" name="Honored"/>
+    <marker id="61598b8d-fe90-47d1-b3da-d177ca379b2f" name="Military"/>
+    <marker id="3b61f2ca-ef39-4d34-b4e1-f6c09e483477" name="Political"/>
+  </markers>
+  <cards>
+    <card id="a88f2213-7711-4699-a94e-23bf10ceedd6" name="First Player Token" size="first_player_token">
+      <property name="Type" value="First Player Token"/>
+    </card>
+    <card id="4c4f1d22-f2e8-46ff-8446-9aa6ec0a45a6" name="Honor Dial" size="honor_dial">
+      <property name="Type" value="Honor Dial"/>
+      <alternate name="Honor Dial" type="2">
+        <property name="Type" value="Honor Dial"/>
+      </alternate>
+      <alternate name="Honor Dial" type="3">
+        <property name="Type" value="Honor Dial"/>
+      </alternate>
+      <alternate name="Honor Dial" type="4">
+        <property name="Type" value="Honor Dial"/>
+      </alternate>
+      <alternate name="Honor Dial" type="5">
+        <property name="Type" value="Honor Dial"/>
+      </alternate>
+    </card>
+    <card id="b57c595e-d5ae-4fba-82c8-954a0b78c4a8" name="Imperial Favor" size="imperial_favor">
+      <property name="Type" value="Imperial Favor"/>
+      <property name="Bonus Military Skill" value="+1"/>
+      <alternate name="Imperial Favor" type="Political">
+        <property name="Type" value="Imperial Favor"/>
+        <property name="Bonus Military Skill" value=""/>
+        <property name="Bonus Political Skill" value="+1"/>
+      </alternate>
+    </card>
+    <!--
```

Comment:
Did these mean to get commented out? Trying to set up a game fails to load them atm.

Reply:
huh i actually have no idea what I was doing there, I'll check it out later